src/app-config/credentials.ts

```diff
@@ -1,12 +1,14 @@
-import path from 'path';
 import fs from 'fs-extra';
+import path from 'path';
 import AppConfig from './config';
 
 const CREDENTIALS_FILENAME = 'creds.json';
 
 interface Credential {
   account: string;
   password: string;
+  id: string;
+  type: string;
```

Comment:
where are id/type used?

Reply:
They're not used, but they were required here to avoid a Typescript error, or using `Promise<any>` as a return type instead

Reply:
Just needed an export on the Credential interface. 

This was using the node internal Credential class
https://github.com/architect-team/architect-cli/pull/102/commits/70784ed74a59d1bc1d040be1e39291c1024df41f#diff-6ad7c92ac31697eb55fb1be46d6d36c2R54